packages/turf-bezier/index.js

```diff
@@ -33,9 +33,9 @@ var Spline = require('./spline.js');
  * };
  *
  * var curved = turf.bezier(line);
+ * curved.properties = { stroke: '#0f0' };
  *
  * //addToMap
- * curved.properties = { stroke: '#0f0' };
  * var addToMap = [line, curved]
  */
```

Comment:
There's two parts, the map which uses the addToMap as a featureCollection with all the properties. The second is the JSDocs example being rendered on the website which does not include the addToMap details.
Most of the extra properties added is to properly visualize the map example.

Hopefully that helps answer your question